components/PackagesVersions.vue

```diff
@@ -19,41 +19,46 @@
 </template>
 
 <script setup>
+  const cloud_store = use_cloud_store()
+  const { is_running } = storeToRefs(cloud_store)
+
   const props = defineProps({
-    route_prefix: { type: String, required: true },
+    schema: { type: Object, required: true },
   })
-  const { route_prefix } = props
-
-  const cloud_store = use_cloud_store()
+  const { schema } = props
 
   const packages_versions = ref([])
 
-  async function get_packages_versions() {
-    await api_fetch(
-      `${route_prefix}/versions`,
-      { method: "GET" },
+  function get_packages_versions() {
+    api_fetch(
+      { schema },
       {
         response_function: (response) => {
+          console.log("response", response)
           packages_versions.value = response._data.versions
         },
       },
     )
   }
 
-  watch(cloud_store.is_running, (value) => {
+  watch(is_running, (value) => {
+    console.log("watch", value)
     if (value === true) {
       get_packages_versions()
     }
   })
 
   onMounted(() => {
-    if (cloud_store.is_running === true) {
+    console.log("mounted", is_running)
+    if (is_running.value === true) {
```

Comment:
I think you can do directly
```suggestion
    if (is_running.value) {
```